### PR TITLE
chore(stories): use consistent widget import

### PR DESCRIPTION
**Pull request by Haroenv**
**Summary**

- use consistent widget import in stories
- makes sure that widgets used from the instantsearch module aren't typed as `any`
- changes the extension of all remaining files to TS

**Result**

all widgets are instantiated from the instantsearch module provided by withHits, and all story files are in TS

### CODE REVIEW

**Comment by eunjae-lee** [comment]
Can you explain to me the difference between importing from `../src/...` and `instantsearch.xxx.yyy` here? Is it for consistency or is there any functional difference?

**Comment by Haroenv** [comment]
It's mostly for consistency, but also a little to make sure that we test what's the public api (the umd bundle) @eunjae-lee

File: .storybook/decorators/withHits.ts [hunk 1]
```diff
@@ -5,14 +5,16 @@ import defaultPlayground from '../playgrounds/default';
...
+
 export const withHits = (
   storyFn: ({
     container,
     instantsearch,
     search,
   }: {
     container: HTMLElement;
-    instantsearch: any;
+    instantsearch: InstantSearchUMDModule;
```
Comment (Haroenv) [inline .storybook/decorators/withHits.ts:17]: yes, this adds typings @francoischalifour

**Review by eunjae-lee — APPROVED** [review]
Looks great!

**Pull request merged**